addons/stock/views/product_views.xml

```diff
@@ -168,17 +168,13 @@
                     </div>
                     <field name="show_qty_update_button" invisible="1"/>
                     <label for="qty_available" class="oe_inline" invisible="not is_storable" groups="stock.group_stock_user"/>
-                    <div class="d-flex gap-2 align-items-baseline hover-show" invisible="not is_storable" groups="stock.group_stock_user">
-                        <field name="qty_available" style="max-width:100px;" readonly="True" groups="!stock.group_stock_manager"/>
-                        <field name="qty_available" style="max-width:100px;" readonly="show_qty_update_button" groups="stock.group_stock_manager"/>
+                    <div class="o_row" invisible="not is_storable" groups="stock.group_stock_user">
+                        <a id="update_product_qty_button" type="object"
+                           name="action_open_quants" groups="stock.group_stock_manager"><field name="qty_available" readonly="True"/></a>
```

Comment:
We want to keep the current behavior of editing qty_available. Just modify the action to open the wanted view